### ISSUE TITLE
Microsoft Word: announce diagram type for diagrams instead of "embeded object"

### ISSUE THREAD

**Issue by Adriani90** [nvaccess/nvda#8564]
### Steps to reproduce:
1. Create a 2x4 table
2. Fill the first column with a till d and the second with one till four
3. create a line Chart

### Actual behavior:
NVDA announces the Chart as "embeded object" when navigating through the word document.

### Expected behavior:
NVDA should announce the diagram type when navigating through the document.

#### NVDA Installed/portable/running from source:
Both

#### NVDA version:
2018.2.1.

#### Windows version:
Windows 8.1

#### Name and version of other software in use when reproducing the issue:
MS Office 2010

#### Does the issue still occur after restarting your PC?
yes

#### Have you tried any other versions of NVDA?
no

**Comment by seanbudd** [comment]
@Adriani90 - is this still an issue with the latest versions of the software and NVDA?

**Comment by Adriani90** [comment]
This issue is still valid. Current test results with NVDA 2025.2 and MS Office MSO (Version 2507 Build 16.0.19029.20136) 64 Bit

When using UIA in MS Word:
* NVDA reports "Diagram embeded object" when moving with arrow keys through the document and focusing the diagram object, but diagram type (e.g. pie or line) is not reported.
* Pressing enter while focusing the diagram object adds linefeed character instead of moving the focus into the diagram to navigate the values.

When using object model:
* NVDA reports" Diagram" when focusing the diagram with arrow keys, but does not report the diagram type (pie, line etc.)
* Pressing enter on a diagram object moves the focus into the diagram and NVDA can navigate its values same as in Microsoft Excel. This feature has been introduced in #7046 but is not available when using UIA.
